src/main/snapshot.ts

```diff
@@ -379,7 +385,10 @@ export async function processSnapshot(
       mixpanel
     );
     snapshotPaths.push(stateSnapshotPath);
-    removeUselessStore(storePath);
+    await removeUselessStore(storePath).catch((error) => {
+      if (error.code === "ENOENT" || error.code === "ENOTDIR") return; // Ignore if not exist
+      console.error("Error while removing useless store: ", error);
+    });
```

Comment:
```suggestion
    try
    {
      await removeUselessStore(storePath);
    }
    catch (error)
    {
      if (error.code === "ENOENT" || error.code === "ENOTDIR") return; // Ignore if not exist
      console.error("Error while removing useless store: ", error);
    }
```

but it doesn't seem to a necessary work since it's a hotfix.

Reply:
While reviewing the comment I've noticed that `Promise.all` prevents other removals to be performed when a single one fails, so I've updated the PR accordingly.